orangecontrib/timeseries/widgets/owtabletotimeseries.py

```diff
@@ -66,9 +66,10 @@ def set_data(self, data):
         if data.domain.has_continuous_attributes():
             vars = [var for var in data.domain.variables if isinstance(var, TimeVariable)] + \
                    [var for var in data.domain.variables
-                    if var.is_continuous and not isinstance(var, TimeVariable)]
+                    if var.is_continuous and not isinstance(var,
+                                                            TimeVariable)] + \
```

Comment:
`and not var.is_time`? Can also `var.is_time` on the instance check above.

Reply:
If `TimeVariable` is a `ContinuousVariable`, could we not simply
```
vars = [var for var in data.domain.variables if var.is_continuous] + \
                   [var for var in data.domain.metas if var.is_continuous]
```

Reply:
Yes. This was to make variables appear grouped.

Reply:
Ah, right. I'll stick to that. I think it is nice, having them grouped.